DSharpPlus/Entities/User/DiscordPresence.cs

```diff
@@ -93,8 +93,8 @@ public DiscordGuild Guild
         internal DiscordPresence(DiscordPresence other)
         {
             this.Discord = other.Discord;
-            this.Activity = other.Activity;
-            this.RawActivity = other.RawActivity;
+            this.Activity = new DiscordActivity(other.Activity);
+            this.RawActivity = new TransportActivity(this.Activity);
```

Comment:
This isn't really cloning the raw activity, its "cloning" a `DiscordActivity`. It's unable to clone every field:

(Left is original, right is "clone")

![rider64_2021-10-02_19-45-08](https://user-images.githubusercontent.com/39029839/135728480-72e5f303-ffce-4aac-bc34-af953ece0d5a.png)

 Shall I introduce a new constructor, that properly clone the raw activity?